### ISSUE TITLE
init and concatenate str error

### ISSUE THREAD

**Issue by zltnzltn** [yassineselmi/homeassistant-azure-tts#1]
Hi, i got two errors with your integration:
my configuration.yaml is:

```
    #https://github.com/yassineselmi/homeassistant-azure-tts
  - platform: azure_tts
    service_name: tts_microsoft_noemi_notok
    cache: false
    api_key: ####################
    language: hu-HU
    gender: Female
    #type: hu-HU-NoemiNeural
    type: NoemiNeural
    rate: 100
    volume: 100
    pitch: default
    contour: (0, 0) (100, 100)
    region: westeurope
```
my automation is:

```
alias: Announcement, Time (Microsoft)
description: ''
trigger:
  - platform: time_pattern
    minutes: /15
condition: []
action:
  - service: tts.tts_microsoft_noemi_notok
    data:
      entity_id: media_player.living_room_speaker, media_player.bedroom_speaker
      message: {{ now().hour}} óra {{ "%0.02d" | format(now().strftime("%-M") | int) }} perc
mode: single
```

Error1
```
Error on init TTS: No TTS from azure_tts for 'message: 20 óra 30 perc'
8:30:51 PM – (ERROR) Text-to-Speech (TTS)

Logger: homeassistant.components.tts
Source: components/tts/__init__.py:188
Integration: Text-to-Speech (TTS) (documentation, issues)
First occurred: 8:30:51 PM (1 occurrences)
Last logged: 8:30:51 PM

Error on init TTS: No TTS from azure_tts for 'message: 20 óra 30 perc'
```

Error2
```
Error occurred for Azure TTS: can only concatenate str (not "bytes") to str
8:30:51 PM – (ERROR) azure_tts (custom integration)

Logger: custom_components.azure_tts.tts
Source: custom_components/azure_tts/tts.py:415
Integration: azure_tts (documentation, issues)
First occurred: 8:30:51 PM (1 occurrences)
Last logged: 8:30:51 PM

Error occurred for Azure TTS: can only concatenate str (not "bytes") to str
```
do you have a solution for this issue?

also id like to change the ptch of the voice a bit deeper, and at sample site (microsoft) and in azur, its posible to change this attribute. id like to use 0.9 for pitch and 1.2 for speed

Thanks,
Zoltan

ps:
with his integration it works:
  [https://github.com/georgezhao2010/azure_cognitive_speech](https://github.com/georgezhao2010/azure_cognitive_speech)
```
  - platform: azure_cognitive_speech
    service_name: tts_microsoft_noemi
    cache: false
    api_key: #############
    region: westeurope
    default_voice: Noemi
```

**Comment by zltnzltn** [comment]
hi, any idea about this?

**Comment by yassineselmi** [comment]
Fixed in PR #2
